stories/cosmoz-input.stories.js

```diff
@@ -53,6 +53,7 @@ export const contour = () => html`
 			--cosmoz-input-line-display: none;
 			--cosmoz-input-contour-size: 1px;
 			--cosmoz-input-background: white;
+			--cosmoz-input-opacity: 0.7;
```

Comment:
I don't think we want to have the inputs featured in this story be semi-transparent.

Reply:
Yes, but this is the whole point of the change. According to [mockups](https://www.figma.com/file/eczucze3bcnenVmnPDnbrK/action-dialog?type=design&node-id=0-1&mode=design&t=SAe332WbgGfLQmzu-0) and the UX meetings, the inputs in the forms should all be in a lighter color. 

I thought of setting the color variable directly, but the risk I foresee is that the disabled input characteristics might not be obvious anymore, so I went for a general layer of opacity

Reply:
Opacity is not the correct way to make things lighter. You have to set each color to a lighter, solid color. You should never use opacity in this way. We can merge this PR as it might come useful in certain situations, but not for this purpose.